setup.py

```diff
@@ -3,48 +3,86 @@
 from setuptools import setup, find_packages
 
 
-setup(
-    name='py-evm',
-    # *IMPORTANT*: Don't manually change the version here. Use the 'bumpversion' utility.
-    version='0.2.0-alpha.14',
-    description='Python implementation of the Ethereum Virtual Machine',
-    long_description_markdown_filename='README.md',
-    author='Piper Merriam',
-    author_email='piper@pipermerriam.com',
-    url='https://github.com/ethereum/py-evm',
-    include_package_data=True,
-    py_modules=['evm', 'trinity', 'p2p'],
-    install_requires=[
+deps={
+    'evm': [
         "aiohttp>=2.3.1,<3.0.0",
         "async_lru>=0.1.0,<1.0.0",
+        "coincurve>=7.0.0,<8.0.0",
         "cryptography>=2.0.3,<3.0.0",
         "cytoolz>=0.9.0,<1.0.0",
         "eth-bloom>=1.0.0,<2.0.0",
+        "eth-hash[pycryptodome]",
         "eth-utils>=1.0.1,<2.0.0",
+        "plyvel==1.0.4",
```

Comment:
same, should not be part of the base `py-evm` install.

Reply:
Moved it into new `evm-extras` section (see longer comment below)